### PR TITLE
Fix: Get value.length while value is null

### PR DESCRIPTION
**Pull request by hongquan**

### CODE REVIEW

File: src/autocomplete.js [hunk 1]
```diff
@@ -534,6 +534,9 @@ yourlabs.Autocomplete.prototype.hide = function() {
 // This function is in charge of hilighting the right result from keyboard
 // navigation.
 yourlabs.Autocomplete.prototype.move = function(e) {
+	if (this.value === null)
```
Comment (jpic) [inline src/autocomplete.js]: Could you fix indentation for this line please ?

We use 4 spaces for indentation.

Comment (hongquan) [inline src/autocomplete.js]: Ok. I've updated. Sorry for copy/paste error :smile_cat:

**Comment by jpic** [comment]
In which use case did you notice this ?

**Comment by hongquan** [comment]
I used it in my Django app. I notice that when I put the cursor to input box, I get an error in JavaScript console (Firebug), saying that "value is null".

But now, after rolling back the fix, I cannot reproduce it.

**Comment by hongquan** [comment]
Update: Now I can reproduce by accident

selection_207

**Pull request merged**